validation-test/Reflection/reflect_empty_struct_compat.swift

```diff
@@ -0,0 +1,75 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-build-swift -lswiftSwiftReflectionTest -I %S/Inputs/EmptyStruct/ %s -o %t/reflect_empty_struct -target x86_64-apple-macosx10.15.0
+// RUN: %target-codesign %t/reflect_empty_struct
+// RUN: %target-run %target-swift-reflection-test %t/reflect_empty_struct | %FileCheck %s --check-prefix=CHECK-%target-ptrsize --dump-input fail
+
+// RUN: %target-build-swift -lswiftSwiftReflectionTest -I %S/Inputs/EmptyStruct/ %s -o %t/reflect_empty_struct -target x86_64-apple-macosx10.14.0
+// RUN: %target-codesign %t/reflect_empty_struct
+// RUN: %target-run %target-swift-reflection-test %t/reflect_empty_struct | %FileCheck %s --check-prefix=CHECK-%target-ptrsize --dump-input fail
+
+// REQUIRES: objc_interop
+// REQUIRES: executable_test
+// REQUIRES: OS=macosx
+
+import SwiftReflectionTest
+
+import EmptyStruct
+
+@_alignment(1) struct EmptyStruct { }
```

Comment:
Why are you declaring the alignment here?

Reply:
The compiler always generates `BuiltinTypeDescriptor`s for types with declared alignment (because the normal layout algorithm would mis-align them), so this is just to exercise that code path.